index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Scheherazade+New&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+    <title>African Devs</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div class="nav-buttons">
+
+                <ul>
+                    
+                    <li>
+                        <a href="#projects">Projects</a>
+                    </li>
+                    <li>
+                        <a href="#skills">Skills</a>
+                    </li>
+                    <li>
+                        <a href="#contacts">Contacts</a>
+                    </li>
+    
+                </ul>
+            </div>
+
+            <div id="github_image">
```

Comment:
Replace the div element with an HTML 5 semantic element like the section element. A semantic element clearly describes its meaning to both the browser and the developer. This will make it easier for search engines as well as make your code more readable.

Reply:
- [ ] Great review you have here😉 I'd suggest you add some emojis to your reviews to make it friendlier 👍